mir-ci/mir_ci/fixtures/servers.py

```diff
@@ -90,3 +92,19 @@ def mir_demo_server():
 @server
 def miriway():
     return snap("miriway", channel="stable", classic=True)
+
+
+@server(
+    ServerCap.ALL
+    ^ (ServerCap.FLOATING_WINDOWS | ServerCap.DRAG_AND_DROP | ServerCap.DISPLAY_CONFIG | ServerCap.SCREENCOPY_BANDWIDTH)
+)
+def gnome_shell():
+    return deb(
+        "gnome-shell",
+        "--wayland",
+        "--no-x11",
+        "--wayland-display",
+        DisplayServer.get_wayland_display(),
+        debs=("gnome-shell",),
```

Comment:
This is the default (the first argument).
```suggestion
```

I've optimized this for usage (so you don't have to say `debs=…` when your deb is the same as your command.